src/app/config.rs

```diff
@@ -79,6 +79,9 @@ pub struct Config {
     /// Which config provider to use to load configs into memory
     pub config_provider_mode: ConfigProviderMode,
 
+    // How frequently to poll redis for config updates when using the redis config provider
```

Comment:
```suggestion
    /// How frequently to poll redis for config updates when using the redis config provider
```

https://doc.rust-lang.org/reference/comments.html#doc-comments